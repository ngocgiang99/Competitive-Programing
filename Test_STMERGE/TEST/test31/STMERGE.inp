10
4 10
418097638 240205248 901213294 718790129 891399281 514990289 321877383 862746539 618201677 783233548 
209442020 131818568 127350641 703038965 254239387 831908462 957747216 279531345 831294208 449663746 
709573280 48994597 524479145 991567656 15035863 948678839 513998442 226297926 38837416 429719946 
401178968 623854592 693433417 683377509 216136905 193029180 597038638 508614732 617508433 75312415 
4 2
511807942 348359437 
514258397 944661650 
648671036 634895020 
540601807 190004539 
2 7
790762074 705243524 577592768 496519376 581655092 942602931 276282936 
721206239 18910800 817509452 798367621 794815411 359910884 497850899 
5 8
364057189 212977167 674284596 888160124 735947144 869909636 815433984 681474555 
648738113 507494904 247508341 619001428 24776135 514055982 97489896 179724137 
767682957 694966572 715663599 481844322 433681738 223224120 340036958 378658466 
388708337 310261974 881541707 781416399 606401208 873502155 568971530 849738701 
840194836 883702812 834017072 261204674 402571962 723483662 324931731 286757651 
9 4
879861575 947847524 458629112 216989994 
376756130 256957595 593954361 112838173 
633818168 440231900 429895449 4433990 
857128926 842636430 17129867 24177961 
581344170 662888019 354843846 171804811 
886241008 74107155 284883005 584380017 
662000453 339952369 232519590 554941217 
54838911 934577235 168540406 858783006 
462906546 827083229 999418466 726392111 
8 1
479446110 
565639255 
908724121 
833517888 
78116769 
486360541 
646483383 
946766754 
6 7
814094241 72333633 464051212 445239381 849115311 362669570 570466782 
802705168 492711027 298143241 414199370 76583868 985096388 107568060 
15269189 346104543 245873591 443981247 207544068 572679997 344039839 
293857529 900267004 478366682 169297684 933726348 149883618 254951421 
403259624 717207520 282013867 347043627 251265586 980612248 122465419 
427400999 904430150 636261530 96395257 166621687 815047354 193910794 
2 9
341281634 800638286 925898894 112376681 16470100 484619019 836463072 531084115 69060580 
783810085 395818259 212033702 546375847 671813058 240192732 734874279 725156723 913909063 
6 5
725315420 458762840 698154564 469182722 288014263 
174405851 61956236 37835560 580321818 95672564 
183615679 264693134 636506202 264318729 313312976 
389604539 946935946 207356863 264527605 538675059 
507147521 328429563 822356387 872291515 967189076 
369727201 812114831 968191449 679324502 861343702 
10 4
535595807 127048833 902182197 5734375 
355277057 834712773 953840537 788006049 
649652883 710208199 882097294 46766789 
563275178 830645998 822695435 264963799 
231631449 841563100 999525443 998692764 
915557112 104802937 542675884 224250393 
944816153 702667752 299663099 186809748 
425261203 255807375 86340389 286906256 
377233274 842749714 965351263 15860180 
552988708 706532952 406288158 298444388 
