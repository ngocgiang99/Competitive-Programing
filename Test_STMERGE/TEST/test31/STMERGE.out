323591851
348359437
276282936
543781326
216989994
479446110
270220610
69060580
288014263
5734375
